src/main/java/christmas/constants/menu/Menu.java

```diff
@@ -0,0 +1,54 @@
+package christmas.constants.menu;
+
+import static christmas.constants.menu.MenuType.APPETIZER;
+import static christmas.constants.menu.MenuType.DESSERT;
+import static christmas.constants.menu.MenuType.DRINKS;
+import static christmas.constants.menu.MenuType.MAIN;
+import static christmas.exception.ErrorCode.INVALID_MENU_ORDER;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public enum Menu {
+    MUSHROOM_SOUP(APPETIZER, "양송이수프", 6000),
+    TAPAS(APPETIZER, "타파스", 5500),
+    SALAD(APPETIZER, "시저샐러드", 8000),
+    STAKE(MAIN, "티본스테이크", 55_000),
+    LIB(MAIN, "바비큐립", 54_000),
+    PASTA_(MAIN, "해산물파스타", 35_000),
+    PASTA(MAIN, "크리스마스파스타", 25_000),
+    CAKE(DESSERT, "초코케이크", 15_000),
+    ICE_CREAM(DESSERT, "아이스크림", 5000),
+    ZERO_COLA(DRINKS, "제로콜라", 3000),
+    WINE(DRINKS, "레드와인", 60_000),
+    CHAMPAGNE(DRINKS, "샴페인", 25_000);
+
+    private final MenuType menuType;
+    private final String name;
+    private final int price;
+
+    Menu(MenuType menuType, String name, int price) {
+        this.menuType = menuType;
+        this.name = name;
+        this.price = price;
+    }
+
+    public static Menu from(String input) {
```

Comment:
단순히 `input`이라는 이름보단 어떤 입력인지를 더 자세히 적어줄 수 있을 것 같아요
```suggestion
    public static Menu from(String menuName) {
```